README.md

```diff
@@ -517,16 +517,19 @@ Now we'll sign the RPM:
 
 ```
 rpm --resign <rpm>.rpm  # --addsign would allow us to apply multiple signatures to the RPM
-rpm -qi<file.rpm>  # should now show that the file is signed
+rpm -qi <file.rpm>  # should now show that the file is signed
 rpm -Kv  # should contain NOKEY errors in the lines containing Signature
 # This is because the the (public) key of the RPM signing key is not present,
 # and must be added to the RPM client config to verify the signature:
-sudo rpm --import <publicKey>.asc
+sudo rpm --import <publicKey>.asc  # Keys are in the lfs repos
 rpm -Kv  # Signature lines will now contain OK instead of NOKEY
+rpm qi <file.rpm> | grep -oP "key ID \K\w+"" | xargs gpg -k  # Verify the RPM was signed with the correct key
 ```
 
 You can then proceed with distributing the package, via the "test" or "prod" repo,
 as appropriate.
+   
+:warning: Remember that you hard-coded a GPG key ID in your `~/.rpmmacros` file to be used any time you sign an RPM from your Fedora machine/VM. If you have to switch between keys, you'll want to either update that file or use a different environment for the other key.
```

Comment:
@zenmonkeykstop - The most helpful catch will be the added step above around running `rpm qi <file.rpm> | grep -oP "key ID \K\w+"" | xargs gpg -k` to verify the RPM was signed with the expected key, but this general warning also should help remind folks that the id is hardcoded in that file. Does this seem like an adequate improvement to how we currently have no correct key check step for the signer?

Reply:
actually, since we are moving the docs, I'll close this PR and move it into the wiki